README.md

```
@@ -1,19 +1,119 @@
+- **Datadog/Zabbix**: 大規模システムの監視設計・実装
+- **Custom Scripts**: Python/Bashによる運用自動化スクリプト開発
+- **Security**: WAF導入、脆弱性診断の実施
+
+### 🚀 Recent Achievements
+- AIを活用した自動コードレビューシステムの導入
+- IoT Core + Lambda + DynamoDBを使用したリアルタイムデータ処理基盤の構築
+- 同一AWSアカウント内の本番/開発環境を分離し、セキュリティを強化
+
+## 📈 Activity Graph
+
+![](https://github-readme-activity-graph.vercel.app/graph?username=zukizukizuki&theme=tokyo-night&hide_border=true)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)